index.js

```diff
@@ -118,6 +120,14 @@ const parseArgs = ({
     }
   );
 
+  const checkSafeOptionValue = (optionAsUsed, longOptionName, value) => {
```

Comment:
```suggestion
  const checkSafeOptionValue = (rawOption, longOption, value) => {
```
**suggestion (non-blocking):** Thoughts on `rawOption` over `optionAsUsed` and `longOption` to match other utilities?

Reply:
I'll revisit that. (I renamed after I got confused about whether it still had leading dashes for error message.)

Reply:
Back to `longOption`, which I had started with! 
I have switched `optionAsUsed` to `dashedOption`, since my main concern was how to use it in the message.